src/app/about/team/Boards/TeamMemberCard/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { type StaticImageData } from 'next/image';
+
+import Heading from '#components/Heading';
+import ImageWrapper from '#components/ImageWrapper';
+import Link from '#components/Link';
+
+import styles from './styles.module.css';
+
+export interface Props {
+    image: StaticImageData | string;
+    alt: string;
+    name?: string;
+    designation?: string;
+    showDescription?: boolean;
+    link?: string;
+}
+
+export default function TeamMemberCard(props: Props) {
+    const {
+        image,
+        alt,
+        name,
+        designation,
+        showDescription,
+        link,
+    } = props;
+
+    if (link) {
```

Comment:
Link here is necessary, so lets remove this logic and just have one render.

Reply:
In the member Details Section i have used this component where it need div instead of link.